### ISSUE TITLE
Compile failure if the crate using the `wasm_init!` macro does not itself depend on wasm_bindgen.

### ISSUE THREAD

**Issue by kyren** [returnString/wasm-init#1]
I don't know how to solve this, but the `#[wasm_bindgen]` proc macro does not appear to support renaming the `wasm_bindgen` crate to something else, and refers to the module path `wasm_bindgen` in the expanded output. This causes `wasm_init!` to not compile if the caller does not itself depend on wasm_bindgen.

I implemented a replacement for the `inventory` crate using `ctor` and `wasm-init`. Calling `inventory::submit!` without depending on `wasm_bindgen` produces get the following output:

```
error[E0433]: failed to resolve: use of undeclared crate or module `wasm_bindgen`
  --> src/core/src/scripting/globals.rs:39:1
   |
39 | / inventory::submit!(GlobalLib::new("serde global markers", |ctx| {
40 | |     scripting::serde::global_markers(ctx);
41 | |     Ok(())
42 | | }));
   | |___^ use of undeclared crate or module `wasm_bindgen`
   |
   = note: this error originates in the attribute macro `$crate::__macrodeps::wasm_bindgen` which comes from the expansion of the macro `inventory::submit` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0425]: cannot find value `FUNCTION` in this scope
  --> src/core/src/scripting/globals.rs:39:1
   |
39 | / inventory::submit!(GlobalLib::new("serde global markers", |ctx| {
40 | |     scripting::serde::global_markers(ctx);
41 | |     Ok(())
42 | | }));
   | |___^ not found in this scope
   |
   = note: this error originates in the attribute macro `$crate::__macrodeps::wasm_bindgen` which comes from the expansion of the macro `inventory::submit` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0405]: cannot find trait `WasmDescribe` in this scope
  --> src/core/src/scripting/globals.rs:39:1
   |
39 | / inventory::submit!(GlobalLib::new("serde global markers", |ctx| {
40 | |     scripting::serde::global_markers(ctx);
41 | |     Ok(())
42 | | }));
   | |___^ not found in this scope
   |
   = note: this error originates in the attribute macro `$crate::__macrodeps::wasm_bindgen` which comes from the expansion of the macro `inventory::submit` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0425]: cannot find function `inform` in this scope
  --> src/core/src/scripting/globals.rs:39:1
   |
39 | / inventory::submit!(GlobalLib::new("serde global markers", |ctx| {
40 | |     scripting::serde::global_markers(ctx);
41 | |     Ok(())
42 | | }));
   | |___^ not found in this scope
   |
   = note: this error originates in the attribute macro `$crate::__macrodeps::wasm_bindgen` which comes from the expansion of the macro `inventory::submit` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by kyren** [comment]
I have a solution incoming...

**Comment by returnString** [comment]
Fixed in 0.2.2
